source/simulate/run-simulation.md

```diff
@@ -150,310 +149,208 @@ Let's also say you have the following `data.csv` and that you want to calculate
 person_id,person_salary,person_age
 1,2694,40
 2,2720,43
-3,1865,45
-4,1941,23
-5,2393,31
-6,3008,47
-7,2286,23
-8,3386,28
-9,2929,38
-10,3981,37
-11,3643,38
-12,2078,23
+3,3865,45
+4,1300,23
+5,0,12
+6,0,14
+7,2884,44
+12,1200,38
+8,3386,27
+9,2929,28
+10,0,10
+11,1600,35
 ```
 
+In the following example, we will use the [pandas](https://pandas.pydata.org) library to iterate over the data.
```

Comment:
Are we iterating? Maybe we should just omit it so to not confuse the averted user.

Reply:
You are right, we are not precisely iterating.
Removed.